### PR TITLE
Add ssb docker and documentation

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: project/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM alpine
+MAINTAINER Benedict Lau <benedict.lau@groundupworks.com>
+
+RUN apk add --update-cache
+RUN apk add autoconf automake build-base libtool ncurses nodejs nodejs-npm python
```
Comment (benhylau) [inline project/docker/Dockerfile]: This can definitely be improved to cut down 70% of image size. Now it's > 300 MB.

File: project/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+Getting Started with ScuttleMesh
+================================
+
+**ScuttleMesh** is [Secure Scuttlebutt](https://github.com/ssbc/secure-scuttlebutt) over a self-configuring encrypted mesh network formed by Raspberry Pis.
+
```
Comment (patcon) [inline project/README.md]: I common pattern I use is to give a quick one-liner on the main tech used, to help orient people minimally without sending them away into linked websites.

What about these as options?
- [x] **ScuttleMesh**Secure Scuttlebutt over a self-configuring encrypted mesh network formed by Raspberry Pis.
- [ ] **Secure Scuttlebutt (SSB).** A database of unforgeable append-only feeds, optimized for efficient replication for peer to peer protocols.
- [ ] **Mesh Orange.** A small ramdisk system running modern Debian to support mesh networking on ARM single-board computers.

File: project/docs/ssb-docker-examples.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    },
+    "signature": "6joMmp7Wg1+hOJ50iFbbragrNt+9ksWhyMTZ8bQ2vU4KetP0EbWq7J8d/Su4yAeVVWBcD4Z+WsWvfYmZaYRZCQ==.sig.ed25519"
+  },
+  "timestamp": 1527037940321
+}
+```
+
+## `sbot gossip.peers`
+
+This happens over local network discover with three Raspberry Pis plugged into a home router via ethernet cables. No ssb pub is involved here.
```
Comment (patcon) [inline project/docs/ssb-docker-examples.md]: > over local network discover

typo? missing word?

File: project/docs/sbot-install-on-sbc.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+Installing Scuttlebot on SBCs
```
Comment (patcon) [inline project/docs/sbot-install-on-sbc.md]: Spell out SBC the first time?

File: project/docs/ssb-docker-examples.md (hunk 3 above)
Comment (benhylau) [reply project/docs/ssb-docker-examples.md]: `discovery` :) will fix

**Comment by benhylau** [comment]
Thanks for the review @patcon don't forget to submit entry to take.md to participate in our first funding cycle :)

**Pull request merged**